localisation/000_Other_title_names.csv

```diff
@@ -1,349 +1,351 @@
-d_stormriggers_adj;Stormrigger;;;;;;;;;;;;;x
-d_highborne_company;Highborne Company;;Hochgebohrenen-Kompanie;;;;;;;;;;;x
-d_highborne_company_adj;Highborne;;hochgebohrenen;;;;;;;;;;;x
-d_centaur_company;Centaur Company;;Zentauren-Kompanie;;;;;;;;;;;x
-d_centaur_company_adj;Centaur;;zentaurisch;;;;;;;;;;;x
-d_farraki_company;Farraki Company;;Farraki-Kompanie;;;;;;;;;;;x
-d_farraki_company_adj;Farraki;;farrakisch;;;;;;;;;;;x
-d_zandalari_company;Zandalari Company;;Zandalari-Kompanie;;;;;;;;;;;x
-d_zandalari_company_adj;Zandalari;;zandalarisch;;;;;;;;;;;x
-d_korkron;Kor'kron;;Kor'kron;;;;;;;;;;;x
-d_korkron_adj;Kor'kron;;kor'kronisch;;;;;;;;;;;x
+e_valarjar;Halls of Valor;Salles des Valeureux;Hallen der Tapferkeit;;Cámaras del Valor;;;;;;;;;x
+d_roughnecks;Roughnecks;Baroudeurs;Raubeine;;Bellacos;;;;;;;;;x
+d_roughnecks_adj;Roughnecks;Baroudeuse;Raubeine;;Bellacos;;;;;;;;;x
+d_ravenholdt;Ravenholdt;Ravenholdt;Rabenholdt;;Ravenholdt;;;;;;;;;x
+d_ravenholdt_adj;Ravenholdt;Ravenholdt;Rabenholdt;;Ravenholdt;;;;;;;;;x
+d_hemets_hunters;Hemet's Hunters;Chasseurs d'Hemet;Hemets Jäger;;Cazadores de Hemet;;;;;;;;;x
+d_hemets_hunters_adj;Hemet's Hunters;Chasseurs d'Hemet;Hemets Jäger;;Cazadores de Hemet;;;;;;;;;x
+d_bloodsail_pirates;Bloodsail Pirates;Voile sanglante;Blutsegelpiraten;;Piratas Velasangre;;;;;;;;;x
+d_bloodsail_pirates_adj;Bloodsail;Sanglante;blutsegel;;Velasangre;;;;;;;;;x
+d_dogs_of_war;Dogs of War;Chiens de guerre;Hunde des Krieges;;Perros de la guerra;;;;;;;;;x
+d_dogs_of_war_adj;War;Caninoguerrière;kriegshündisch;;;;;;;;;;;x
+d_stormriggers;Stormriggers;Perce-orage;;;Armadores de tormentas;;;;;;;;;x
+d_stormriggers_adj;Stormrigger;Perce-orage;;;Armadore de tormentas;;;;;;;;;x
+d_highborne_company;Highborne Company;Compagnie bien-né;Hochgebohrenen-Kompanie;;Compañía Altonato;;;;;;;;;x
+d_highborne_company_adj;Highborne;bien-né;hochgebohrenen;;Altonato;;;;;;;;;x
+d_centaur_company;Centaur Company;Compagnie centaure;Zentauren-Kompanie;;Compañía Centauro;;;;;;;;;x
+d_centaur_company_adj;Centaur;centaure;zentaurisch;;Centauro;;;;;;;;;x
+d_farraki_company;Farraki Company;Compagnie farraki;Farraki-Kompanie;;Compañía Farraki;;;;;;;;;x
+d_farraki_company_adj;Farraki;farraki;farrakisch;;Farraki;;;;;;;;;x
+d_zandalari_company;Zandalari Company;Compagnie zandalari;Zandalari-Kompanie;;Compañía Zandalari;;;;;;;;;x
+d_zandalari_company_adj;Zandalari;zandalari;zandalarisch;;Zandalari;;;;;;;;;x
+d_korkron;Kor'kron;Kor'kron;Kor'kron;;Kor'kron;;;;;;;;;x
+d_korkron_adj;Kor'kron;kor'kron;kor'kronisch;;Kor'kron;;;;;;;;;x
 d_taunka_band;Winterhorns;;;;;;;;;;;;;x
 d_taunka_band_adj;Winterhorn;;;;;;;;;;;;;x
 d_vrykul_company;Drakujar;;;;;;;;;;;;;x
 d_vrykul_company_adj;Drakujar;;;;;;;;;;;;;x
-d_drakkari_band;Drakkari Band;;;;;;;;;;;;;x
-d_drakkari_band_adj;Drakkari;;;;;;;;;;;;;x
-d_wolvar_band;Wolvar Band;;;;;;;;;;;;;x
-d_wolvar_band_adj;Wolvar;;;;;;;;;;;;;x
+d_drakkari_band;Drakkari Band;Bande drakkari;;;Compañía Drakkari;;;;;;;;;x

```

Comment:
I'd add Drakkari-Kompanie, drakkarisch, Wolvar-Kompanie and wolvarisch

Reply:
I'll put this in the new push, thank you.